scripts/powershell/setup.ps1

```diff
@@ -0,0 +1,87 @@
+$ErrorActionPreference = "Stop"
+
+function Select-Clone-Method
+{
+    Write-Host @"
+Select a cloning method.
+
+Press 1 for HTTPS
```

Comment:
optional: may help to say what the URI would look like here, because some people may not know what "ssh" means in this context